python/ray/_raylet.pyx

```diff
@@ -4307,14 +4310,10 @@ cdef class CoreWorker:
         return (CCoreWorkerProcess.GetCoreWorker().GetWorkerContext()
                 .CurrentActorMaxConcurrency())
 
-    def get_queued_future(self, task_id: Optional[TaskID]) -> asyncio.Future:
+    def get_queued_future(self, task_id: Optional[TaskID]) -> ConcurrentFuture:
```

Comment:
Looks like it is okay to just use concurrent futures here, so I used it

Reply:
It is also better becasue the concurernt future is thread-safe whereas the asyncio future is not.